component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -2005,14 +2005,6 @@ public String getContextIdentifier(HttpServletRequest request) {
         return request.getParameter("sessionDataKey");
     }
 
-    /**
-     * Check ID token in EmailOTP OAuth.
-     */
-    @Override
-    protected boolean requiredIDToken(Map<String, String> authenticatorProperties) {
```

Comment:
we shouldn't remove protected methods. Let's remove the override tag and mark it as deprecated.

Reply:
ACK

Reply:
we can add      @Deprecated annotation as well.